book_source/source/module-1-introduction-to-python/lesson-3-strings-and-lists/string-functions.md

```diff
@@ -33,7 +33,7 @@ Here is a shortlist of useful functions for `str` in Python.
 |s.lower()      |Returns a new string that is the lowercase version of s                                                                                                                                  |
 |s.upper()      |Returns a new string that is the uppercase version of s                                                                                                                                  |
 |s.find(target) |Returns the index of the first occurrence of target in s. If not found, returns -1.                                                                                                      |
-|s.strip()      |Returns a new string that has all the leading and trailing whitespace removed.Also has functions lstrip() and rstrip() that remove only left whitespace or right whitespace respectively.|
+|s.strip()      |Returns a new string that has all the leading and trailing whitespace removed. Also has functions lstrip() and rstrip() that remove only left whitespace or right whitespace respectively.|
```

Comment:
Nit: Do you think you could go to the method reference tables and add code formatting to the left-hand sides that show syntax?